server/server.js

```diff
@@ -95,7 +95,7 @@ module.exports = function(port, googleAuthoriser) {
     function getTweetsWithHashtag() {
         var query = {
             q: hashtags.join(" OR "),
-            since_id: sinceIdH
+            since_id: sinceIdH,
```

Comment:
There's not really a strict need, but it makes it easier to edit - I wrote that line on my branch before pulling the fix.